app/models/pipeline_output.rb

```diff
@@ -23,15 +23,6 @@ def generate_report
     end
   end
 
-  def distinct_genuses
```

Comment:
I don't understand the choice of removing this and adding more complexity to  taxononomy_details function